aries_cloudagent/wallet/routes.py

```diff
@@ -757,6 +786,90 @@ async def wallet_set_did_endpoint(request: web.BaseRequest):
         return web.json_response({"txn": transaction.serialize()})
 
 
+@docs(tags=["wallet"], summary="Create a EdDSA jws using did keys with a given payload")
+@request_schema(JWSCreateSchema)
+@response_schema(WalletModuleResponseSchema(), description="")
+async def wallet_jwt_sign(request: web.BaseRequest):
+    """
+        Request handler for jws creation using did.
+
+    Args:
+        "headers": { ... },
+        "payload": { ... },
+        "did": "did:example:123",
+        "verificationMethod": "did:example:123#keys-1"
+        with did and verification being mutually exclusive.
+    """
+    context: AdminRequestContext = request["context"]
+    body = await request.json()
+    did = body.get("did")
+    if did is None:
+        did = body.get("verificationMethod").split("#", 1)[0]
+    if did is None:
+        raise web.HTTPBadRequest(reason="did and verificationMethod required.")
+    payload = body["payload"]
+    headers = body.get("headers", {})
+    async with context.session() as session:
+        wallet = session.inject_or(BaseWallet)
+        if not wallet:
+            raise web.HTTPForbidden(reason="No wallet available")
```

Comment:
I think it's a bit silly to do `inject_or` for this since we should always have a wallet but I see that you're doing as the Romans do so I think this is fair.